README.md

```diff
@@ -250,6 +250,18 @@ The username used to authenticate.
 
 The password used to authenticate.
 
+##### `sasl_oauth_token_provider`
+
+The Token Provider class.
```

Comment:
Can you improve this documentation with a realistic example? Also, the value passed is not in fact a class, but an object.

Also, does `ssl_ca_certs_from_system` need to be `true` for this to work?